tests/utils.js

```diff
@@ -702,13 +703,14 @@ module.exports = {
    *                                       api logs, if value equals 'sentinel', will watch sentinel logs instead.
    * @return {Promise}        completion promise
    */
-  updateSettings: (updates, ignoreReload) => {
+  updateSettings: (updates, ignoreReload,wdio) => {
     const watcher = ignoreReload &&
       Object.keys(updates).length &&
       waitForSettingsUpdateLogs(ignoreReload);
 
     return updateSettings(updates).then(() => {
       if (!ignoreReload) {
+        if(wdio){return;}
```

Comment:
Running your test locally It seems you can pass `ignoreReload` as true and your tests pass. Any reason to keep the `wdio` check? If so can you convert it as mentioned in the global above?